### ISSUE TITLE
ping: unknown host archive.matrix.org

### ISSUE THREAD

**Issue by bkil** [matrix-org/matrix-public-archive#273]

**Comment by Mikaela** [comment]
> We are responding to reports that the archive is indexing on rooms where the ops might reasonably expect it not to be. Currently, we believe it is doing the right thing, but have taken the service down out of an abundance of caution while we can look into it more closely.

- [`@neilj:matrix.org` at `#matrix-public-archive:matrix.org`](https://matrix.to/#/%23matrix-public-archive%3Amatrix.org/%24BDin6ZXpIZ2or4BrzziMqUSP6YSnv2Ijl7nBgxl50e0?via=matrix.org&via=evulid.cc&via=element.io)

They also said:

> I'd like to sync with the relevant team (they are in US timezone), but yes I would expect us to merge https://github.com/matrix-org/matrix-public-archive/pull/239 or mitigate the problem in some other way that achieves the same thing.
> We'll apply the same spirit to all protocols, so we'll consider this in the context of IRC and XMPP (and others). 

Related: #47

**Comment by bkil** [comment]
Just for completeness sake, here's a reference that you can open without having to install anything:

* https://view.matrix.org/room/!SzoPnANsRYxITaDPaJ:matrix.org/?anchor=$1ZSaJxRQJGUnZIyQaAmaM-tvhJEREyqfXiHb79gsgE4#:~:text=07:08:31
* https://archive.matrix.org/r/matrix-public-archive:matrix.org/date/2023/06/26?at=$QZ0FcBhlc_vWohP8oxGp69xDlgFUuq4H4_9amZdxSd0

**Comment by Mikaela** [comment]
This issue seems to be resolved.

```
% LANG=en ping -c10 archive.matrix.org
PING archive.matrix.org(2606:4700:10::6814:c925 (2606:4700:10::6814:c925)) 56 data bytes
64 bytes from 2606:4700:10::6814:c925 (2606:4700:10::6814:c925): icmp_seq=1 ttl=55 time=17.3 ms
64 bytes from 2606:4700:10::6814:c925 (2606:4700:10::6814:c925): icmp_seq=2 ttl=55 time=18.5 ms
64 bytes from 2606:4700:10::6814:c925 (2606:4700:10::6814:c925): icmp_seq=3 ttl=55 time=20.7 ms
64 bytes from 2606:4700:10::6814:c925 (2606:4700:10::6814:c925): icmp_seq=4 ttl=55 time=23.7 ms
64 bytes from 2606:4700:10::6814:c925 (2606:4700:10::6814:c925): icmp_seq=5 ttl=55 time=19.9 ms
64 bytes from 2606:4700:10::6814:c925 (2606:4700:10::6814:c925): icmp_seq=6 ttl=55 time=28.7 ms
64 bytes from 2606:4700:10::6814:c925 (2606:4700:10::6814:c925): icmp_seq=7 ttl=55 time=20.4 ms
64 bytes from 2606:4700:10::6814:c925 (2606:4700:10::6814:c925): icmp_seq=8 ttl=55 time=20.0 ms
64 bytes from 2606:4700:10::6814:c925 (2606:4700:10::6814:c925): icmp_seq=9 ttl=55 time=21.8 ms
64 bytes from 2606:4700:10::6814:c925 (2606:4700:10::6814:c925): icmp_seq=10 ttl=55 time=21.1 ms

--- archive.matrix.org ping statistics ---
10 packets transmitted, 10 received, 0% packet loss, time 9013ms
rtt min/avg/max/mdev = 17.262/21.218/28.668/2.987 ms
```

**Comment by wojtekLs** [comment]
@Mikaela indeed. Host is reachable (from server machine) but site is  still showing placeholder.
![Zrzut ekranu 2023-06-28 o 11 49 21](https://github.com/matrix-org/matrix-public-archive/assets/109670049/727a8e73-68bc-4588-80b7-5878b5945202)

**Comment by Mikaela** [comment]
Blocked by: https://github.com/vector-im/element-meta/issues/1807

-----

I hid my comments that predated the issue being written.
